mm/2s2h/Enhancements/Player/UnderwaterOcarina.cpp

```diff
@@ -0,0 +1,21 @@
+#include <libultraship/bridge.h>
```

Comment:
Recently we stopped trying to include the `libultraship/bridge.h` header as it basically includes almost everything from LUS, increasing compile times for files that don't need it.
This can be replaced with 
```suggestion
#include "public/bridge/consolevariablebridge.h"
```